templates/results.html

```diff
@@ -3,8 +3,39 @@
 {% set active_page = "results" %}
 
 {% block content %}
-    <h1>Results</h1>
-    <p> Welcome to the Results page!</p>
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Initiate Data Analysis</title>
+</head>
+<body>
+  <button id="analyzeButton">Initiate Data Analysis</button>
+
+  <script>
```

Comment:
Extract this to another file, and link it to the HTML (like in #61)

Reply:
> Extract this to another file, and link it to the HTML (like in #61)
Sir i am done with the corections you recommended. You can now verify